CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.1 (2020-04-09)
+
+- Fixed the **package.json** `peerDependencies` to require a minimum of `^@skyux/core@3.4.2`. [#76](https://github.com/blackbaud/skyux-lookup/pull/76) (Thanks [@jeffbdye](https://github.com/jeffbdye)!)
```

Comment:
Looking at Jeff's PR, it looks like the version should be `"^3.14.2"` Am I missing something?

Reply:
And is smashing `@skyux/core` after the `^` our standard style? I'm wondering if this should be "... to require `@skyux/core` versions `3.14.2` or later."

Reply:
Also, do we need to reference that this was in the package.json file? I don't feel like we usually do that. Seems like usually we'd just say "Fixed `peerDependencies` to require..." But I can't remember for sure.